shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx

```diff
@@ -471,7 +459,7 @@ In this implementation, you initiate and remove  <Vg k="ENGINE" /> when the app
 
 2. **Start <Vpd k="PRODUCT" />**
 
-    Replace `Start` with the following code:
+    In your script file, replace the `Start` method with the following:
```

Comment:
replace `Start` with ... 

This is the last time I will comment, please do the others yourself.